packages/shopify-api/docs/reference/auth/tokenExchange.md

```diff
@@ -0,0 +1,78 @@
+# shopify.auth.tokenExchange
+
+Begins the OAuth process by exchanging the current user's [Session Token](#ZL:TODO) for an [Access Token](#ZL:TODO).
+
+Learn more:
+- [Token Exchange](../../guides/oauth.md#token-exchange)
+
+## Examples
+
+### Node.js
+```ts
+app.get('/auth', async (req, res) => {
+  const shop = shopify.utils.sanitizeShop(req.query.shop, true)
+  const headerSessionToken = getSessionTokenHeader(request);
+  const searchParamSessionToken = getSessionTokenFromUrlParam(request);
+  const sessionToken = (headerSessionToken || searchParamSessionToken)!;
+
+  await shopify.auth.exchangeToken({
```

Comment:
This should be `shopify.auth.tokenExchange`.

https://github.com/Shopify/shopify-app-js/blob/4d47824256e46ff8e87f26ea58919a85989a7279/packages/shopify-app-remix/src/server/authenticate/admin/strategies/token-exchange.ts#L129-L133